chapter-06/Exercise_08/RGBcolors.scala

```diff
@@ -0,0 +1,19 @@
+/*8. Write an enumeration describing the eight corners of the RGB color cube. As
+IDs, use the color values (for example, 0xff0000 for Red ).*/
+
+package Exercise_08
+
+object RGBcolors extends Enumeration {
+
+  type RGBcolors = Value
+  val Black = Value(0x000000,"Black")
+  val Magenta = Value(0xff00ff, "Magenta")
+  val White = Value(0xffffff,"White")
+  val red = Value(0xff0000,"Red")
```

Comment:
Tem algum motivo pra você não ter deixado com R maiúsculo? Em Scala, constantes são expressas em `CamelCase`.
```suggestion
  val Red = Value(0xff0000,"Red")
```

Reply:
o Shift que traiu!!!